docs/override-configs.md

```diff
@@ -1,3 +1,29 @@
+## Enabling/disabling components
+
+To enable/disable components in Ozone, you need to modify the configurations in the [docker-compose-files.txt](https://github.com/ozone-his/ozone-docker-compose/blob/main/scripts/docker-compose-files.txt) in the ozone-docker-compose for your Ozone deployment. Here's how you can do it:
+
+**Access the excutable files**
+
+Navigate to the docker-compose-files.txt file in the ozone-docker-compose.
+
+**Enable/disable a component**
+
+To enable a component, you need to provide it's excutable file in the ozone-docker-compose.
+
+To disable a component, you need to remove it's excutable file in the ozone-docker-compose.
+
+**Restart Ozone Services**
+
+After making the necessary changes to enable or disable components, restart the Ozone services. Run the command `./stop-demo.sh` to stop the services, followed by `./start-demo.sh` to start them again with the updated configuration.
+
+**Verify changes**:
+
+Once the services are restarted, verify that the desired components are now enabled or disabled as intended. Access the Ozone components and check for the presence or absence of the components based on your configuration changes.
```

Comment:
Running a `docker compose -p ozone ps` command could be a good way to see if the services are correctly started.

Reply:
Even better:
`docker compose -p ozone ps --format "table {{.Name}} \t {{.Image}} \t {{.Status}}"`